core/src/main/java/com/linecorp/armeria/server/annotation/RequestObject.java

```diff
@@ -26,7 +26,11 @@
  *
  * @see RequestConverterFunction
  * @see RequestConverter
+ * @deprecated No more putting {@link RequestObject} on the parameter which needs to be converted by one
```

Comment:
No more putting .. -> `There's no need to put {@link RequestObject} annotation anymore on a parameter converted by a {@link RequestConverterFunction}.`